components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html

```diff
@@ -56,17 +56,18 @@
               <chef-td (click)="filterFor('manager_type', node.manager)">{{ node.manager }}</chef-td>
               <chef-td (click)="filterFor('last_contact', node.last_contact)">{{ node.last_contact }}</chef-td>
               <chef-td>
-                <chef-icon class="status-icon" [ngClass]="node.scan_data.status"
+                <chef-icon [attr.id]="node.name" class="status-icon--pointer" [ngClass]="node.scan_data.status"
```

Comment:
in order for chef-tooltips to show, the "target" or thing you hover over needs to have an `id` that matches the `for` attribute on the chef-tooltip.

Reply:
i also added a small pointer utility class that you can choose to do away with if you want - just giving the cursor a pointer when hovering the icon

Reply:
thank you!! that's the 🔑 -- thank you so much!

do we have a place where our components are documented - like in the code or anything? i didn't actually take the time to look (i should have) but wondering for my future self.

Reply:
cd into components/chef-ui-library
npm install
npm start
navigate to localhost:3333

There is pretty solid documentation for each component there!